src/repr/src/scalar.rs

```diff
@@ -205,6 +205,27 @@ impl TryFrom<Datum<'_>> for Option<f32> {
     }
 }
 
+impl TryFrom<Datum<'_>> for OrderedFloat<f32> {
+    type Error = ();
+    fn try_from(from: Datum<'_>) -> Result<Self, Self::Error> {
```

Comment:
At some point we should start marking all the `try_from` fns as `#[inline]`. Feel free to add it to this PR, but don't feel obliged!

Reply:
Sounds good; will do in a separate PR